src/Settings/SettingsAttribute/CustomRankingAttribute.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * This file is part of AlgoliaSearch Client PHP Helper.
+ *
+ * (c) Algolia Team <contact@algolia.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Algolia\AlgoliaSearch\Helper\Settings\SettingsAttribute;
+
+use Algolia\AlgoliaSearch\Helper\Contracts\SettingContract;
+use Algolia\AlgoliaSearch\Helper\Helpers\Str;
+
+// @internal
+final class CustomRankingAttribute implements SettingContract
+{
+    /**
+     * @var array<int, string>
+     */
+    private static $customRankingKeys = [
+        '*ed_at',
+        'count_*',
+        '*_count',
+        'number_*',
+        '*_number',
+    ];
+
+    /**
+     * Checks if the given key/value is a 'CustomRanking'.
+     *
+     * @param string                         $key
+     * @param null|array<int, string>|string $value
+     * @param array<int, string>             $customRanking
+     *
+     * @return array<int, string>
+     */
+    public function getDetectedSettings($key, $value, $customRanking)
```

Comment:
Can you rename `$customRanking` by `$setting`? Let's keep arguments naming just like the contract.

Reply:
I agree with you